### DISCUSSION TITLE
Daily 04/12/2023

### DISCUSSION THREAD

**Discussion by lucasdpt** [Sinan-Karakaya/R-Type#31] [Daily]
Start at 9h30

**Comment by JasonLAGOUTE** [comment]
Hey ! Aujourd'hui ca va découper le projet et se partager les tâches. Commencer a integrer le serveur dans le runtime.

**Reply by jovanHillion** [reply]
Yo ajd je commence l'implémentation du scipting

**Comment by lucasdpt** [comment]
Suite de la lib réseau + Découpage des tâches

**Comment by Sinan-Karakaya** [comment]
Intégration du Runtime à l'éditeur
